### PR TITLE
Publish as ESM

### PR DESCRIPTION
**Pull request by adamraine**
This needs to be done in a breaking release, because this package is currently published as CJS and this PR changes the published JS to be ESM.

This will ensure Lighthouse and Chrome launcher are using the same version of lighthouse logger in their shared `lighthouse-logger` package.

### CODE REVIEW

**Review by connorjclark — APPROVED** [review]

File: test/chrome-launcher-test.ts [hunk 1]
```diff
@@ -38,7 +41,7 @@ const launchChromeWithOpts = async (opts: Options = {}) => {
   }
 };
 
-describe('Launcher', () => {
+describe.only('Launcher', () => {
```
Comment (brendankenny) [inline test/chrome-launcher-test.ts:44]: from debugging?

File: tsconfig.json [hunk 2]
```diff
@@ -10,12 +11,14 @@
...
+        "noUnusedParameters": true,
+        "allowSyntheticDefaultImports": true,
     },
     "exclude": [
         "node_modules"
     ],
     "include": [
-        "src"
+        "src",
+        "types"
```
Comment (brendankenny) [inline tsconfig.json:22]: does it really need the d.ts file of itself?

File: test/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../tsconfig.json",
+  "ts-node": {
+    "files": true,
+  },
+  "include": [
+    "*-test.ts",
+    "../types"
+  ]
+}
```
Comment (brendankenny) [inline test/tsconfig.json:10]: nit: newline

File: test/chrome-launcher-test.ts (hunk 1 above)
Comment (adamraine) [reply test/chrome-launcher-test.ts:44]: Explains why the tests were passing

File: tsconfig.json (hunk 2 above)
Comment (adamraine) [reply tsconfig.json:22]: Of itself? The d.ts file is for lighthouse logger

Comment (brendankenny) [reply tsconfig.json:22]: lol, I've been looking too long at lighthouse-logger I guess :)

**Pull request merged**